app/utilities/first_time_data_generator.rb

```diff
@@ -25,6 +25,91 @@ def seed_demo_data(user)
     # Do nothing
     return unless team
 
+    # create custom repository samples
+    repository = Repository.create(
+      name: 'Samples',
+      team: team,
+      created_by: user
+    )
+
+    # create list value column for sample types
+    repository_column_sample_types = RepositoryColumn.create(
+      repository: repository,
+      created_by: user,
+      data_type: :RepositoryListValue,
+      name: 'Sample Types'
+    )
+
+    # create list value column for sample groups
+    repository_column_sample_groups = RepositoryColumn.create(
+      repository: repository,
+      created_by: user,
+      data_type: :RepositoryListValue,
+      name: 'Sample Groups'
+    )
+
+    # create few list items for sample types
+    repository_items_sample_types = []
+    ['Potato leaves', 'Tea leaves', 'Potato bug'].each do |name|
+      item = RepositoryListItem.create(
+        data: name,
+        created_by: user,
+        last_modified_by: user,
+        repository_column: repository_column_sample_types,
+        repository: repository
+      )
+      repository_items_sample_types << item
+    end
+
+    # create few list items for sample groups
+    repository_items_sample_groups = []
+    %i(Fodder Nutrient Seed).each do |name|
+      item = RepositoryListItem.create(
+        data: name,
+        created_by: user,
+        last_modified_by: user,
+        repository_column: repository_column_sample_groups,
+        repository: repository
+      )
+      repository_items_sample_groups << item
+    end
+
+    repository_rows_to_assign = []
+    # Generate random custom respository sample names and assign sample types
+    # and groups
+    # (0...3).map{65.+(rand(26)).chr}.join << '/'
```

Comment:
Remove the comment.

Reply:
probably we will do that when we will remove Samples completely